pyproject.toml

```diff
@@ -49,7 +49,7 @@ _ = "returns.contrib.hypothesis._entrypoint"
 python = "^3.9"
 
 typing-extensions = ">=4.0,<5.0"
-mypy = { version = ">=1.10,<1.11", optional = true }
+mypy = "1.11.0"
```

Comment:
```suggestion
mypy = mypy = { version = ">=1.11,<1.12", optional = true }
```

Reply:
You would also need to run `poetry update` again

Reply:
done